### PR TITLE
feat: editable PDP

### PR DESCRIPTION
**Pull request by agurtovoy**
https://github.com/user-attachments/assets/25a98f70-2e69-4d8b-8617-1fb3f618871f

### CODE REVIEW

File: core/app/[locale]/(default)/product/[slug]/page.tsx [hunk 1]
```diff
@@ -5,11 +5,11 @@ import { getFormatter, getTranslations, setRequestLocale } from 'next-intl/serve
 import { Suspense } from 'react';
 
 import { IconBlock } from '@/vibes/soul/sections/icon-block';
-import { AccordionItem, ProductDescription } from '@/vibes/soul/sections/product-description';
 import { ProductDetail } from '@/vibes/soul/sections/product-detail';
 import { Field } from '@/vibes/soul/sections/product-detail/schema';
 import { pricesTransformer } from '~/data-transformers/prices-transformer';
 import { LocaleType } from '~/i18n/routing';
+import { AccordionItem, ProductDescription } from '~/makeswift/components/product-description';
```
Comment (agurtovoy) [inline core/app/[locale]/(default)/product/[slug]/page.tsx:12]: Import switch.

File: core/app/[locale]/(default)/product/[slug]/page.tsx [hunk 2]
```diff
@@ -327,6 +327,7 @@ export default async function Product({ params: { locale, slug }, searchParams }
       <ProductDetail action={addToCart} fields={formattedFields} product={formattedProduct} />
 
       <ProductDescription
+        product={formattedProduct}
```
Comment (agurtovoy) [inline core/app/[locale]/(default)/product/[slug]/page.tsx:330]: Passing product so we can properly scope the snapshot/label the changes for the Publish dialog.

File: core/app/[locale]/(default)/product/[slug]/page.tsx [hunk 3]
```diff
@@ -332,6 +332,7 @@ export default async function Product({ params: { locale, slug }, searchParams }
           src: product.defaultImage?.url ?? '',
           alt: product.defaultImage?.altText ?? '',
         }}
+        product={formattedProduct}
```
Comment (agurtovoy) [inline core/app/[locale]/(default)/product/[slug]/page.tsx:335]: Passing product so we can properly scope the snapshot/label the changes for the Publish dialog.

**Review by apledger — COMMENTED** [review]
LGTM!

**Review by apledger — APPROVED** [review]

**Pull request merged**